lib/kwtool/crawlers/schemas/keyword.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Kwtool.Crawlers.Schemas.Keyword do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  schema "keywords" do
+    field :phrase, :string
+    field :raw_result, :string
+    field :status, :integer
+    timestamps()
```

Comment:
I think we should add a line break before this one.

Reply:
Updated! Thanks 👍